ViewModelKit/UICollectionReusableView/VMKCollectionReusableHeaderView.h

```diff
@@ -0,0 +1,24 @@
+//
+//  VMKCollectionReusableHeaderView.h
+//  ViewModelKit
+//
+//  Created by Peter Darbey on 08/06/2020.
+//  Copyright © 2020 Andre Trettin. All rights reserved.
```

Comment:
Maybe not really true to put my name here.....

Reply:
I guess this was auto generated